include/mata/afa.hh

```diff
@@ -37,6 +37,14 @@
 
 namespace Mata
 {
+/**
+ * Alternating Finite Automata including structures, transitions and algorithms.
+ *
+ * In particular, this includes:
```

Comment:
Is it the file structured like this? If it is, I would consider adding comments introducing Structure section, etc. directly to code.

Reply:
Beware, that the comment applies to namespace, not file.

Reply:
I am OK with this comment.

Reply:
Ok, I guess that doxygen documentation will benefit from this where all elements of namespace are grouped? Than it's ok.

Reply:
Yes. My bad, I should've made it clear, that this PR mainly applies to doxygen documentation.